config/application.php

```diff
@@ -61,7 +61,7 @@
 Config::define('DB_HOST', env('DB_HOST') ?: 'localhost');
 Config::define('DB_CHARSET', 'utf8mb4');
 Config::define('DB_COLLATE', '');
-$table_prefix = env('DB_PREFIX') ?: 'wp_';
+$table_prefix = (env('DB_PREFIX') || env('DB_PREFIX') === '') ? env('DB_PREFIX') : 'wp_';
```

Comment:
What about just `empty(env('DB_PREFIX')) ? env('DB_PREFIX') : 'wp_';`

Reply:
Looks good to me, I'll get this into a patch tomorrow morning so this is wrapped up.